backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5718830_sys_gh17549_SkipHouseKeeping_SysConfig.sql

```diff
@@ -0,0 +1,4 @@
+-- 2024-03-07T18:50:20.737Z
+UPDATE AD_SysConfig SET ConfigurationLevel='S', Name='de.metas.housekeeping.SkipHouseKeeping',Updated=TO_TIMESTAMP('2024-03-07 19:50:20','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_SysConfig_ID=541712
+;
+
```

Comment:
Delete this and it should still work. This way all tasks get skipped. See this part in `org.adempiere.ad.housekeeping.HouseKeepingService`
![image](https://github.com/metasfresh/metasfresh/assets/32903793/b77d116c-fff9-4863-b7d1-96666729f934)